composer/trainer/trainer.py

```diff
@@ -1020,18 +1038,27 @@ def _determine_autoresume_load_path(
         if save_latest_filename is None:
             raise ValueError(
                 "save_latest_filename must be specified so autoresume knows where to load checkpoints from.")
+        # Check run_name is provided if necessary
         if run_name is None:
-            raise ValueError("run_name must be specified so autoresume knows which run to load from.")
```

Comment:
(See slack), but can we continue to require this argument when using autoresume for the time being?